app/assets/javascripts/discourse/components/site-header.js.es6

```diff
@@ -180,6 +354,18 @@ const SiteHeaderComponent = MountWidget.extend(Docking, {
       }
 
       $panel.width(width);
+      if (animate) {
+        $panel.addClass("animate");
+        $headerCloak.addClass("animate");
+        Ember.run.later(() => {
```

Comment:
This is another event you will want to make sure only happens once and is cleared on destroy (see my earlier comments.)